src/sql/workbench/services/errorMessage/browser/errorMessageDialog.ts

```diff
@@ -109,6 +116,18 @@ export class ErrorMessageDialog extends Modal {
 	private updateDialogBody(): void {
 		DOM.clearNode(this._body!);
 		DOM.append(this._body!, DOM.$('div.error-message')).innerText = this._message!;
+		if (this._instructionText) {
+			let childElement = DOM.$('div.error-instruction-text');
+			childElement.innerText = this._instructionText!;
+			if (this._readMoreLink) {
+				let link = new Link(childElement, {
+					label: this._readMoreLabel,
+					href: this._readMoreLink
+				}, undefined, this._openerService);
+				childElement.appendChild(link.el);
```

Comment:
The Link constructor already appends it to the container : https://github.com/Microsoft/azuredatastudio/blob/26455e9113fb97398b2c86273af64c42d218a71a/src/vs/platform/opener/browser/link.ts#L87

Reply:
It got appended only once so probably second call was a no-op.
I've removed the extra call though.